free-1.md

```diff
@@ -57,12 +136,48 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think around 50 percent of the time would be spent on desigining the language.
+I feel like it will be fairly simple to define terms of the lanugage, especially
+because coding languages for visual art already exist and I would be able to model
+aspects of my language after those. However, I think I'll need to make important 
+design decisions on how the the code can be organized from the user end (i.e. 
+creating classes or separate files). I also think I'll need to decide how simple
+the language will be, either by limiting the vocabulary or by limiting the
+parameters for different objects.
+
+The other half of the time will be spent figuring out how to parse and execute 
+the code, figuring out how the shapes and animations will be represented on the 
+screen. I'll need to do some research on which language would be the best one 
+to implement my DSL and determine what can feasibly be animated. I'll also 
+need to decide what kind of user interface to create to allow the user to see
+the result of their code.
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+I would say this project is quite ambitious, particularly for me because I've
+never done my own coding project before and I don't feel exceptionally proficient
+in any programming language. Also, I don't particularly have much knowledge in this
+domain, so it would require a lot of initial research on my part to figure out what
+would be feasible to implement. So it feels like a big idea because it requires a 
```

Comment:
Given these points, I think this project might end up being a bit too ambitious. If you're able to find a good backend to do all the animations it should be okay, but I suspect that aspect could very rapidly become too complex.